lib/full-site-editing/templates.php

```diff
@@ -71,7 +71,7 @@ function gutenberg_register_wp_theme_taxonomy() {
 
 	register_taxonomy(
 		'wp_theme',
-		array( 'wp_template', 'wp_template_part', 'wp_global_styles' ),
+		array( 'wp_global_styles' ),
```

Comment:
Makes me wonder if we should consider using `set_theme_mod()` to store the `wp_global_styles` post IDs used for Global Styles as well? cc/ @nosolosw

Reply:
This can be looked after, as the global styles data coming from the user hasn't landed in 5.8, it's something plugin-only. For context, we only switched to use taxonomies in GS because folks brought up that it should work the same as the templates https://github.com/WordPress/gutenberg/issues/30191

Reply:
I left this untouched because I'm not too familiar with how global styles works, but I'd guess that if only one global styles post is expected per theme, it could work as a theme mod.

Reply:
For context: while now it only uses one CPT per theme, we may want to explore having more (a different one per template area, per post, per post type, etc).